### PR TITLE
add Who Am I field to create DoWhop form and fix style issues

### PR DESCRIPTION
**Pull request by mike-mjk**
1. Add "Who Am I" field to index, admin, and create html files
2. Make submit button say Add To Marketplace for index and create (left as Create DoWhop for admin.html)
3. Change file upload on admin and create to match index.html
4. Change style on "Add To Marketplace" button on create.html to match index.html
5. Get rid of Title tooltip on create.html and admin.html
6. Intentionally Left tooltip on admin "Who is the Doer" field
7. Noticed that we are not validating on "Why Do This DoWhop" question. Is this intentional? line 27 and line 54 of dowhop-description-form-create.js
8. fixed indentation error on index.html ~lines 210-234
9. add image buttons should be styled with cursor:pointer

### CODE REVIEW

File: admin.html [hunk 1]
```diff
@@ -74,9 +78,26 @@ <h2 class="mdl-card__title-text">Create a new DoWhop</h2>
                   <span class="mdl-tooltip" for="doer-description">Enter multiple doers, seperated, by, Commas</span>
                 </div>
                 <div class="mdl-card__actions">
-                  <input id="dowhop-image-capture1" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
-                  <input id="dowhop-image-capture2" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
-                  <input id="dowhop-image-capture3" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
+                  <div id="create-outer-div">
```
Comment (JohannBillar) [inline admin.html]: I would use id descriptions that are more semantic.

File: create.html [hunk 2]
```diff
@@ -37,7 +37,6 @@ <h2 class="mdl-card__title-text">Create a new DoWhop</h2>
                   <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
```
Comment (JohannBillar) [inline create.html]: What part of the code base is using this create.html file? If we need to use it somewhere, it's missing dependencies.

File: scripts/dowhop-description-form-create.js [hunk 3]
```diff
@@ -25,6 +26,7 @@ function submitNewDoWhopEntry(e) {
       titleDescription.value,
       // whyDescription.value,
```
Comment (JohannBillar) [inline scripts/dowhop-description-form-create.js]: Thanks for noticing this! We should be validating whyDescription field and it it should NOT have been commented out. Please un-comment and fix the typo for 'snap' that you found.  👍

**Comment by JohannBillar** [comment]
"7. Noticed that we are not validating on "Why Do This DoWhop" question. Is this intentional? line 27 and line 54 of dowhop-description-form-create.js" ◀️ Please fix this. Nice find!

**Comment by JohannBillar** [comment]
@omar Can you deploy once this feature is completed and test on Safari mobile.

**Comment by mike-mjk** [comment]
Thanks Johann. I'll make more commits in the future. I'll get the commented out piece, too. I won't be able to start working on this until the afternoon though.

**Comment by omarcodex** [comment]
Good catch, @mike-mjk! Thanks for working on that later today. 

@JohannBillar yes, once it's ready I'll test and go from there!

File: admin.html (hunk 1 above)
Comment (mike-mjk) [inline admin.html]: @JohannBillar are you saying to change create-outer-div to something more meaningful? This is the section where users add images. I could change it to add-images-div. Also the individual image icons have divs named create-div-1, create-div-2, create-div-3. I could change these to something like image-1, image-2, image-3. Let me know if you have better suggestions. (And obviously, I will update the css that points to the original names as well)

File: create.html (hunk 2 above)
Comment (mike-mjk) [inline create.html]: I can't remember the precisely how the create html is used. I think Rae may be the person to discuss this with initially.

File: admin.html (hunk 1 above)
Comment (JohannBillar) [inline admin.html]: Thanks @mike-mjk! Yes, that sounds good. For image-1, etc you might have to add more to make sure they stay unique if they are IDs. 
We inherited some of this code so w'Il keep finding stuff to clean.

File: create.html (hunk 2 above)
Comment (JohannBillar) [inline create.html]: Good point! Lets bring this up on Friday.

**Comment by mike-mjk** [comment]
@omarcodex I changed the div names and this is ready for testing

**Pull request merged**